action.yml

```diff
@@ -43,13 +43,21 @@ runs:
       shell: bash
       working-directory: ${{ github.workspace }}
 
-    - name: Commit and push changes
-      if: ${{ inputs.commit-and-push == 'true' }}
+    - name: Make script executable
+      run: chmod +x ${{ github.action_path }}/scripts/create-pr.sh
+      shell: bash
+
+    - name: Setup git config
       run: |
-        git config user.name github-actions
-        git config user.email github-actions@github.com
-        git add .
-        git commit -m "Auto-commit from GitHub Actions"
-        git push
+        git config user.name 'token-generator-app[bot]'
```

Comment:
I've used this bot account before for other workflows in my repos. Is there a better bought account I should be using?

Reply:
If we want to use our organization bot, you can use the info from here

https://github.com/open-sauced/release/blob/main/release.config.js#L54

Reply:
Tested this with our bot account and all good.

![CleanShot 2024-09-03 at 12 33 53](https://github.com/user-attachments/assets/c360d228-ef57-4af4-802f-5eb17bca1707)